pkgs/games/starsector/default.nix

```diff
@@ -57,7 +57,8 @@ stdenv.mkDerivation rec {
     wrapProgram $out/starsector.sh \
       --prefix PATH : ${lib.makeBinPath [ openjdk ]} \
       --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath buildInputs} \
-      --run 'mkdir -p ''${XDG_DATA_HOME:-~/.local/share}/starsector; cd '"$out"
+      --run 'mkdir -p ''${XDG_DATA_HOME:-~/.local/share}/starsector' \
```

Comment:
`--run` doesn't exist for makeBinaryWrapper, so this will fail.

This is not currently possible to port to makeBinaryWrapper since it
1. creates a directory
2. expands an environment variable at runtime

Reply:
I'm not trying to port this package to makeBinaryWrapper since it doesn't depend on wrapGAppsHook. I just made the `--chdir` transformation global for uniformity.